src/se/terhol/test34/HouseImpl.java

```diff
@@ -0,0 +1,92 @@
+package se.terhol.test34;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+
+/**
+ * @author Tereza Holm
+ */
+public class HouseImpl implements House {
+    Map<Room, Set<Room>> myHouse = new HashMap<>();
```

Comment:
Why the name is `myHouse` and not just a `house`? If there is not any specific reason, I would rename it to not confuse others.

Reply:
Done